acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -22,7 +26,12 @@ def initialize_repo_on_host(host, os)
 enabled=1
 gpgcheck=1
     REPO
+    end
 
+    host['platform'].with_version_codename =~ /^(fedora|el|centos)-(\d+)-(.+)$/
+    variant = (($1 == 'centos') ? 'el' : $1)
+    version = $2
+    unless version == '7'
```

Comment:
@kbarber I am seeing failure in vmware. Definitely needs more investigation. Not sure why the environment would have an impact on this.